inference/core/utils/image_utils.py

```diff
@@ -427,11 +427,9 @@ def np_image_to_base64(image: np.ndarray) -> bytes:
     Returns:
         bytes: The base64 encoded image.
```

Comment:
Originally this method was not returning base64 encoded string (maybe I misunderstand but [documentation](https://docs.python.org/3/library/io.html#io.BytesIO.getvalue) does not mention in-flight base64 encoding).